pkg/api/grpc/metadata/metadata.go

```diff
@@ -15,7 +15,7 @@ package metadata
 
 import (
 	"context"
-
+    "erros"
```

Comment:
typo?

Reply:
actually, why was this added? I don't see this import being used

Reply:
> typo?

Oh not saw this